cmd/server-main.go

```diff
@@ -972,11 +972,37 @@ func serverMain(ctx *cli.Context) {
 	}
 
 	// Initialize users credentials and policies in background right after config has initialized.
+	globalIAMFullyInitialized.Store(true)
```

Comment:
This is a "trap", since the value doesn't represent the state.

The goroutine reading it will not be started before the value is set anyway, so I don't think you need to set it to true before anyway, since the first call to `globalIAMSys.Init` will always set it. Correct?

Reply:
hmm, makes sense